Example Apps/Shared/Images.xcassets/remote_control.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "remote_control_icon.png",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "remote_control_icon-1.png",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "remote_control_icon-2.png",
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
```

Comment:
I only provided you one image because you only need the one image I provided for SDL usage. If it were on device, you'd need different sizes, but on SDL you only need one. You should only set the image in the 1x like the other SDL icons.